include/boost/geometry/extensions/gis/projections/proj/hatano.hpp

```diff
@@ -49,8 +49,8 @@ namespace boost { namespace geometry { namespace projections
             static const int NITER = 20;
             static const double EPS = 1e-7;
             static const double ONETOL = 1.000001;
-            static const double CN = 2.67595;
-            static const double CS = 2.43763;
+            static const double CN_ = 2.67595;
+            static const double CS_ = 2.43763;
             static const double RCN = 0.37369906014686373063;
```

Comment:
This is generated code.
Anyway, I will update the generator (it is not yet in git)